libraries/default.rb

```diff
@@ -0,0 +1,43 @@
+#
+# Cookbook Name:: kernel-modules
+# Author:: Jeremy MAURO <j.mauro@criteo.com>
+#
+# Copyright 2016, Criteo.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module KernelModules
+  # Function for handling the modprobe format
+  def self.modprobe_config(name, config)
+    config.each do |cmd, values|
+      return "#{cmd} #{name}" if values.nil? || values.empty?
```

Comment:
You should not use return here:
1. in ruby we should avoid it
2. return in a block might result in `LocalJumpError`
3. returning in a `each` is not good